docs/can-guides/commitment/recipes/todomvc-with-steal/5-toggle-event/index.html

```diff
@@ -1,47 +1,47 @@
 <!-- index.stache -->
 <section id="todoapp">
-	<header id="header">
-		<h1>{{ this.appName }}</h1>
-		<input id="new-todo" placeholder="What needs to be done?">
-	</header>
-	<section id="main" class="">
-		<input id="toggle-all" type="checkbox">
-		<label for="toggle-all">Mark all as complete</label>
-		<ul id="todo-list">
-			{{# for(todo of this.todosList) }}
-				<li class="todo {{# if(todo.complete) }}completed{{/ if }}">
-					<div class="view">
-						<input 
-							class="toggle" 
-							type="checkbox"
-							{{# if(todo.complete) }}checked{{/ if }}
-							on:click="todo.toggleComplete()"
-						>
-						<label>{{ todo.name }}</label>
-						<button class="destroy"></button>
-					</div>
-					<input class="edit" type="text" value="{{ todo.name }}">
-				</li>
-			{{/ for }}
-		</ul>
-	</section>
-	<footer id="footer" class="">
-		<span id="todo-count">
-			<strong>{{ this.todosList.active.length }}</strong> items left
-		</span>
-		<ul id="filters">
-			<li>
-				<a class="selected" href="#!">All</a>
-			</li>
-			<li>
-				<a href="#!active">Active</a>
-			</li>
-			<li>
-				<a href="#!completed">Completed</a>
-			</li>
-		</ul>
-		<button id="clear-completed">
-			Clear completed ({{ this.todosList.complete.length }})
-		</button>
-	</footer>
+  <header id="header">
+    <h1>{{ this.appName }}</h1>
+    <input id="new-todo" placeholder="What needs to be done?">
+  </header>
+  <section id="main" class="">
+    <input id="toggle-all" type="checkbox">
+    <label for="toggle-all">Mark all as complete</label>
+    <ul id="todo-list">
+      {{# for(todo of this.todosList) }}
+        <li class="todo {{# if(todo.complete) }}completed{{/ if }}">
+          <div class="view">
+            <input 
+              class="toggle" 
+              type="checkbox"
+              {{# if(todo.complete) }}checked{{/ if }}
+              on:click="todo.toggleComplete()"
+            >
+            <label>{{ todo.name }}</label>
+            <button class="destroy"></button>
+          </div>
+          <input class="edit" type="text" value="{{ todo.name }}">
+        </li>
+      {{/ for }}
+    </ul>
+  </section>
+  <footer id="footer" class="">
+    <span id="todo-count">
+      <strong>{{ this.todosList.active.length }}</strong> items left
+    </span>
+    <ul id="filters">
+      <li>
+        <a class="selected" href="#!">All</a>
+      </li>
+      <li>
+        <a href="#!active">Active</a>
+      </li>
+      <li>
+        <a href="#!completed">Completed</a>
+      </li>
+    </ul>
+    <button id="clear-completed">
+      Clear completed ({{ this.todosList.complete.length }})
+    </button>
+  </footer>
```

Comment:
I think everything should be tabs. Maybe it is and it’s not showing up correctly on GitHub.